src/test/bin/quic_gtest.cpp

```diff
@@ -702,6 +704,7 @@ TEST_P(WithFamilyArgs, InterfaceBinding) {
     }
 }
 
+#ifdef QUIC_API_ENABLE_PREVIEW_FEATURES
```

Comment:
This isn't a version negotiation test, AFAICT.  It just ensures the connection still succeeds when the client uses the `QUIC_VERSION_MS_1` version instead of the official versions.

Reply:
Does it depend on the ability to set a different version though? In other words, does it depend on a preview API at all?

Reply:
oh, I see. The old version SetParam was removed in favor of the preview API.